presto-main/src/test/java/com/facebook/presto/execution/TestThriftTaskStatus.java

```diff
@@ -79,12 +78,6 @@
     public static final HostAddress REMOTE_HOST = HostAddress.fromParts("www.fake.invalid", 8080);
     private TaskStatus taskStatus;
```

Comment:
This is immutable for the duration of the test so it can be `static` (and `final`?) and constructed in a `@BeforeTest` method.

Reply:
It can be static. To construct it @BeforeTest, it can't be final. I'll make the change. Thanks